modules/bitgo/test/v2/unit/internal/tssUtils/ecdsa.ts

```diff
@@ -663,6 +690,20 @@ describe('TSS Ecdsa Utils:', async function () {
     };
   }
 
+  async function nockGetBitgoPublicKeyBasedOnFeatureFlags(coin: string, enterprise: string, bitgoGpgKeyPair: openpgp.SerializedKeyPair<string>): Promise<BitgoGPGPublicKey> {
+    const bitgoGPGPublicKeyResponse: BitgoGPGPublicKey = {
+      name: 'irrelevant',
+      publicKey: bitgoGpgKeyPair.publicKey,
+      enterpriseId: 'enterprise id',
```

Comment:
`enterpriseId` here won't match the mocked enterprise id value specified in parameters